File: atcoder.jp/Python (3.8.2)/abc264/abc264_d.py
S = list(input())
N = len(S)

atcoder = "atcoder"
ans = 0
for i in range(N):
    for j in range(N - i - 1):
        if atcoder.index(S[j]) > atcoder.index(S[j + 1]):
            S[j], S[j + 1] = S[j + 1], S[j]
            ans += 1

print(ans)

File: atcoder.jp/C++ (GCC 9.2.1)/abc264/abc264_d.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    string S;
    cin >> S;
    string atcoder = "atcoder";
    vector<int> V(S.size());
    for (int i = 0; i < S.size(); i++) V.at(i) = atcoder.find(S.at(i));

    int ans = 0;
    for (int i = 0; i < S.size(); i++) {
        for (int j = 0; j < S.size() - i - 1; j++) {
            if (V.at(j) > V.at(j + 1)) {
                swap(V.at(j), V.at(j + 1));
                ans++;
            }
        }
    }
    cout << ans << endl;

    return 0;
}